common/docker/configuration.go

```
@@ -0,0 +1,335 @@
+				Interval: 3 * time.Second, //nolint:gomnd
+				Timeout:  3 * time.Second, //nolint:gomnd
+				Retries:  5,               //nolint:gomnd
+			},
+		},
+		HostConfig: &container.HostConfig{
+			PortBindings: nat.PortMap{
+				"26257/tcp": []nat.PortBinding{{HostPort: "26257"}},
+				"8080/tcp":  []nat.PortBinding{{HostPort: "8080"}},
+			},
+			RestartPolicy: container.RestartPolicy{Name: "unless-stopped"},
+			Mounts: []mount.Mount{{Type: mount.TypeVolume, Source: cfg.DockerEnv["CARDINAL_NAMESPACE"],
+				Target: "/var/lib/cockroach"}},
+			NetworkMode: container.NetworkMode(cfg.DockerEnv["CARDINAL_NAMESPACE"]),
+		},
+	}
+}
+
+func CelestiaDevNet(cfg *config.Config) Config {
+	return Config{
+		Name: getCelestiaDevNetContainerName(cfg),
+		Config: &container.Config{
+			Image: "ghcr.io/rollkit/local-celestia-devnet:latest",
+			ExposedPorts: nat.PortSet{
+				"26657/tcp": struct{}{},
+				"26658/tcp": struct{}{},
+				"26659/tcp": struct{}{},
+				"9090/tcp":  struct{}{},
+			},
+			Healthcheck: &container.HealthConfig{
+				Test:     []string{"CMD", "curl", "-f", "http://127.0.0.1:26659/head"},
+				Interval: 1 * time.Second,
+				Timeout:  1 * time.Second,
+				Retries:  20, //nolint:gomnd
+			},
+		},
+		HostConfig: &container.HostConfig{
+			PortBindings: nat.PortMap{
+				"26657/tcp": []nat.PortBinding{{HostPort: "26657"}},
+				"26658/tcp": []nat.PortBinding{{HostPort: "26658"}},
+				"26659/tcp": []nat.PortBinding{{HostPort: "26659"}},
+				"9090/tcp":  []nat.PortBinding{{HostPort: "9090"}},
+			},
+			RestartPolicy: container.RestartPolicy{Name: "on-failure"},
+			NetworkMode:   container.NetworkMode(cfg.DockerEnv["CARDINAL_NAMESPACE"]),
+		},
+	}
+}
+
+func EVM(cfg *config.Config) Config {
+	daBaseURL := cfg.DockerEnv["DA_BASE_URL"]
+	if daBaseURL == "" {
+		daBaseURL = "http://celestia-devnet"
+	}
+
+	daAuthToken := cfg.DockerEnv["DA_AUTH_TOKEN"]
+	if daAuthToken == "" {
+		daAuthToken = ""
```

This will not work without DA Auth Token, so it should return an error.

---

removing the default value on `daAuthToken`, validation should be handled by `validateDALayer` func in `evm` package